guide/how-it-works/human-readable-addresses.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: guide
+title: Human readable addresses
+description: Easy-to-use bitcoin addresses via DNS payment instructions and the lightning address protocol.
+nav_order: 1
+parent: Payment request formats
+grand_parent: How it works
+permalink: /guide/how-it-works/human-readable-addresses/
+main_classes: -no-top-padding
+image: /assets/images/guide/how-it-works/human-readable-addresses/human-readable-addresses-preview.jpg
+
+---
+
+<!--
+
+Editor's notes:
+
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/human-readable-addresses/human-readable-addresses.jpg"
+   retina = "/assets/images/guide/how-it-works/human-readable-addresses/human-readable-addresses@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/human-readable-addresses/human-readable-addresses-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/human-readable-addresses/human-readable-addresses-mobile@2x.jpg"
+   alt-text = "The word bitcoin address written in binary and plain text"
+   width = 1600
+   height = 400
+   layout = "full-width"
+%}
+
+# Human readable addresses
+
+
+[Bitcoin addresses]({{'/guide/glossary/address/' | relative_url}}) and [payment requests]({{'/guide/how-it-works/payment-request-formats/' | relative_url}}) are long and cryptic, and practically impossible to read, memorize, pronounce, understand, and type. Creating a separate format for human-readable addresses is a logical solution to this problem, and there have been multiple attempts at making them a reality.
+
+The benefits to the user experience include:
+
+- Familiarity: A format that feels more human and familiar to users
+- Ease of sharing: A format that is easier to remember and recall and share in a social setting
+
+Note that human-readable addresses should be <b>only</b> used for direct communication. Whenever payment information is communicated digitally (copy & paste, links, QR codes, messaging, etc), regular addresses should be used for best privacy and security.
```

Comment:
This would be impossible to enforce on any level and is also not how it is currently implemented. See Nostr.

Reply:
Sure thing. Just like address re-use, wallets can encourage best practices, but can't enforce them. That's why I wrote `should`. Or am I misunderstanding?

Reply:
mm gotcha - well then maybe then bolden the should and add "ideally".